types/tx_result.go

```diff
@@ -0,0 +1,14 @@
+package types
+
+import (
+	abci "github.com/tendermint/abci/types"
+)
+
+// TxResult contains results of executing the transaction.
+//
+// One usage is indexing transaction results.
+type TxResult struct {
+	Height    uint64                 `json:"height"`
```

Comment:
Yes, please formalize this somewhere and adjust things.

I have way to many `h := height.(uint64)` and `height := h.(int)` lines... Since abci app returns `int64` for block height and tendermint rpc returns `int`.  So when I want to compare a proof with a header....

I don't care much about the choice (though I like anton's reasoning).  But I would really appreciate consistency, even if that doesn't make it in until 0.10, at least let's standardize it all on the `unstable` branch.